src/main/java/com/plivo/api/models/powerpack/NumberPriority.java

```diff
@@ -2,28 +2,28 @@
 
 public class NumberPriority {
 
-  private String country_iso;
+  private String countryISO;
   private Priority priority;
-  private String service_type;
+  private String serviceType;
 
   public NumberPriority(String country_iso, Priority priority, String service_type) {
-    this.country_iso = country_iso;
+    this.countryISO = country_iso;
```

Comment:
This is existing code and will be a breaking change for clients. If you want to include this change in the PR, you should bump up the major version to 6.0.0

Reply:
reverted